plugins/node/opentelemetry-plugin-mysql/src/mysql.ts

```diff
@@ -17,21 +17,19 @@
 import { BasePlugin, isWrapped } from '@opentelemetry/core';
 import { CanonicalCode, Span, SpanKind } from '@opentelemetry/api';
 import * as mysqlTypes from 'mysql';
+import { format } from 'mysql';
```

Comment:
Do not import a concrete implementation from the plugin being loaded or you will cause a circular require. if you need something from mysql, get it from `this._moduleExports`

Reply:
Oh shoot thanks! TIL

Reply:
I seemed to have had to thread through `format` from `_patchQuery` into `getDbStatement`...WDYT?

Reply:
probably fine